src/codegen.cpp

```diff
@@ -357,6 +357,7 @@ static Function *jlboundserrorv_func;
 static Function *jlcheckassign_func;
 static Function *jldeclareconst_func;
 static Function *jlgetbindingorerror_func;
+static Function *jlboundp_func;
```

Comment:
`boundp` is a common lisp function for testing whether a variable is bound: http://www.ai.mit.edu/projects/iiip/doc/CommonLISP/HyperSpec/Body/fun_boundp.html